client/write.go

```diff
@@ -48,3 +53,28 @@ func (c *Client) Write(key, secret string, force bool) error {
 	}
 	return nil
 }
+
+func (c *Client) WriteFile(key, file string, force bool) error {
+
+	fh, err := os.Open(file)
+	if err != nil {
+		return err
+	}
+
+	fdata := make([]byte, 8192)
+	count, err := fh.Read(fdata)
+	if err != nil {
+		return err
```

Comment:
can we prepare a simple unit test for file bigger than 8k?

Reply:
added: https://github.com/AirHelp/treasury/pull/88/commits/d9ee360ce9a91cad056f9579f6c164e3519e36a4